openshift/4.0/templates/Logging/Dockerfile

```diff
@@ -7,14 +7,17 @@ ARG RUN_DEPS='bash curl ca-certificates gettext'
 ARG APP_ROOT=/opt/app
 ARG APP_LOG=/logging/
 
-RUN apk update && apk upgrade && apk add openssh && apk add zip && apk --no-cache add $BUILD_DEPS $RUN_DEPS && \
+RUN apk update && apk upgrade && apk add openssh && apk add --update openssl && apk add --no-cache tzdata && apk add zip && apk --no-cache add $BUILD_DEPS $RUN_DEPS && \
     curl -sLo /tmp/oc.tar.gz https://mirror.openshift.com/pub/openshift-v$(echo $OC_VERSION | cut -d'.' -f 1)/clients/$OC_VERSION/linux/oc.tar.gz && \
     tar xzvf /tmp/oc.tar.gz -C /usr/local/bin/ && \
     rm -rf /tmp/oc.tar.gz 
 
 RUN mkdir -p ${APP_ROOT} && touch ${APP_ROOT}/liveness && chmod a+r ${APP_ROOT}/liveness && touch ${APP_ROOT}/readiness && chmod a+r ${APP_ROOT}/readiness
-COPY ./sidecar-logging.sh ${APP_ROOT}
-RUN chmod a+rx ${APP_ROOT}/sidecar-logging.sh
+
+COPY . ${APP_ROOT}
+
+RUN rm -Rf ${APP_ROOT}/*.yaml && rm -Rf ${APP_ROOT}/*.env 
+RUN chmod a+rx ${APP_ROOT}/sidecar-logging.sh && chmod a+rx ${APP_ROOT}/bin/s3-put && chmod a+rx ${APP_ROOT}/lib/s3-common.sh
```

Comment:
I don't see a command here to add execution permission to **s3-get** (which is part of this PR)
I see that it is not being used at the moment so would recommend to either add execution permissions to it or delete it for now until needed.

Reply:
resolved